index.js

```diff
@@ -29,20 +33,104 @@ cp.parseBlock = function () {
 	return this.finishNode(node, "BlockStatement");
 };
 
+function strToModuleIdentifier(str) {
+	return '$m_' + str.replace(/-(\w)/g, (_, letter) => letter.toUpperCase()).replace(/[^$_a-zA-Z0-9]/g, '_');
+}
+
+function writePrefix (imports) {
+	let transformed = `Object.defineProperty(exports, '__esModule', { value: true });\n`;
```

Comment:
Aren't you exporting just various named functions? If so, then it should work with CommonJS just fine as-is.